src/Widgets/GalleryWidget/GalleryWidget.scss

```
@@ -0,0 +1,142 @@
+@import "src/assets/stylesheets/variables";
+
+.gallery-widget {
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 0 100px 0;
+  position: relative;
+
+  .slick-dots {
+    bottom: -60px;
+    top: auto;
+    left: auto;
+    right: auto;
+    position: absolute;
+    line-height: 0;
+
+    li {
+      border: 5px solid #fff;
+      margin: 0;
+      display: inline-block;
+      width: auto;
+      height: auto;
+
+      button {
+        margin: 0;
+        height: auto;
+        width: 100%;
+        padding: 0;
+
+        &:before {
+          content: "";
+          border: 3px solid $secondary;
+          display: block;
+          width: 100%;
+          max-width: 180px;
+          overflow: hidden;
+          float: none;
+          position: absolute;
+          width: 100%;
+          height: 100%;
+        }
+
+        &.tab {
+          clear: both;
+          display: block;
+        }
+
+        .image {
+          width: 180px;
+          height: 270px;
+          margin: 0;
+        }
+      }
+    }
+
+    li,
+    button:before {
+      margin: 0;
+      padding: 0;
+    }
+  }
+
+  .slick-slide {
```

See my other comment regarding `slickslide` and `slick-slider`:

```suggestion
  .slick-slider {
```

This can then be of cause be combined with the upper `.slick-slider`.